core/unix/signal_linux_x86.c

```diff
@@ -368,7 +369,8 @@ dump_fpstate(dcontext_t *dcontext, kernel_fpstate_t *fp)
         LOG(THREAD, LOG_ASYNCH, 1, "\n");
     }
 #    endif
-    /* ignore padding */
+    /* XXX i#1312: this needs to get extended to AVX-512. */
+    /* Ignore padding */
```

Comment:
If the comment is going to be changed, add the trailing `.` too.

Reply:
Done.